contracts/README.md

```diff
@@ -0,0 +1,180 @@
+# Devnet contracts for development
+
+This folder contains Cairo and Solidity contracts for developement of Devnet.
+If you wish you to check specifically one of the two chains contracts, please check:
+1. `solidity` folder for Ethereum related contracts.
+2. `cairo` folder for Starknet related contracts, and example of how to work with starknet without
+   running a L1 node.
+
+## Full example of e2e testing with Anvil and Devnet
+
+### Setup of the nodes
+Please ensure that you've [anvil](https://book.getfoundry.sh/getting-started/installation) installed (or you can do the same with HardHat, but the commands here are done with anvil).
+```bash
+anvil
+```
+
+For Starknet, ensure you've Devnet compiled and running with the following params:
+```bash
+cargo run -- --seed 42
+```
+
+Now both nodes are running, Devnet for Starkne and Anvil for Ethereum.
```

Comment:
```suggestion
Now both nodes are running, Devnet for Starknet and Anvil for Ethereum.
```

Reply:
Fixed in the updated [README](https://github.com/glihm/starknet-devnet-rs/tree/7dd5d6e0fe8f14aca40ebd562cc3c5d5e16ce873/contracts/l1-l2-messaging)